lib/cartopy/__init__.py

```diff
@@ -108,4 +108,4 @@
 # Commonly used sub-modules. Imported here to provide end-user
 # convenience.
 import cartopy.crs
-import cartopy.feature
+import cartopy.feature  # noqa: F401  (imported but not used)
```

Comment:
Does this comment stop Stickler complaining?

Reply:
Yes, it's a specific indication to the flake8 tester, actually the "pyflakes" component

If anyone is interested ...
It's really not very clearly explained, but after a bit of digging http://flake8.pycqa.org/en/latest/user/error-codes.html
So, "F" numbers are errors from pyflakes, and "E" numbers are from pycodestyle (pep8-successor).